gammapy/datasets/map.py

```diff
@@ -1316,7 +1316,15 @@ def plot_residuals_spatial(
         ax = residuals.plot(ax, **kwargs)
         return ax
 
-    def plot_residuals_spectral(self, ax=None, method="diff", region=None, **kwargs):
+    def plot_residuals_spectral(
+        self,
+        ax=None,
+        method="diff",
+        region=None,
+        kwargs_fit=None,
+        kwargs_safe=None,
```

Comment:
The docstring should be adapted. And the default in lines 1407-10 should be precised.